queue_services/entity-digital-credentials/src/entity_digital_credentials/worker.py

```diff
@@ -69,8 +69,8 @@ async def process_digital_credential(dc_msg: dict, flask_app: Flask):
             f'bc.registry.business.{FilingCore.FilingTypes.CHANGEOFREGISTRATION.value}',
             f'bc.registry.business.{FilingCore.FilingTypes.DISSOLUTION.value}',
             f'bc.registry.business.{FilingCore.FilingTypes.PUTBACKON.value}',
-            'bc.registry.admin.bn',
-            'bc.registry.admin.manual'
+            'bc.registry.business.bn',
+            'bc.registry.business.manual'
```

Comment:
I agree with `bc.registry.business.bn`.  

I think we should replace `bc.registry.admin.manual` with `bc.registry.admin.revoke`.  My thought is we should still distinguish admin based operations.  We'll need to update the cli to support this in some way.